packages/skeleton-svelte/src/lib/components/Tooltip/types.ts

```diff
@@ -40,4 +40,8 @@ export interface TooltipProps extends Omit<tooltip.Context, 'id' | 'open'> {
 	trigger?: Snippet;
 	/** Provide the template contents of the tooltip itself. */
 	content?: Snippet;
+
+	// Events ---
+	/** Handle the tooltip button hover event. */
+	onmouseover?: () => void;
```

Comment:
@phamduylong So I believe the Tooltip hover event is already handled by `onOpenChange()` event. The original feature request I believe specifically called out the ability to have their tooltip trigger take the appearance of a button, and still have `onclick` functionality.

I don't think there's any harm in keeping `onmouseover`, but can you expand this to also include `onclick` as well?

Once added I think this is ready to go!

Reply:
@phamduylong I pinged you on Discord as well so hopefully you see this before you start working on it. But I'm going to go ahead and implement the change above so we can get this PR merged. This is the last item I'm planning to merge before we pause for our release candidate tomorrow.

Expect those changes to appear momentarily.